packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-calendar.tsx

```diff
@@ -163,6 +193,9 @@ export class GuxMonthCalendar {
 
   private changeYear(increment: number): void {
     this.year = (parseInt(this.year) + increment).toString();
+    const month = getYearMonthObject(this.value).month;
```

Comment:
this logic does not work in the case when the month picker has no initial set value. For a component without a set value, try changing the month by clicking the `>` to change the month. On the feature branch you will see in the console that there is an error being logged, because there is no set value

Reply:
I added a fix here to disable the next/previous year buttons if `this.value` is empty.

Reply:
@jason-evans-genesys It doesn't make sense to me to disable the next/previous year buttons if `this.value` is empty. Having an empty value should not constrain a user to only choosing a month in the current year.